Linkpearl/Plugin.cs

```diff
@@ -91,12 +91,20 @@ public sealed class Plugin : IDalamudPlugin {
 
         var camera = CameraManager.Instance->GetActiveCamera();
         if (camera == null) return null;
+        // gotta cast because it's the CS struct vector3 type
+        var cameraPos = (Vector3) camera->CameraBase.SceneCamera.Object.Position;
```

Comment:
Is this good style? Don't we usually leave no space between the type and the expression in a cast?

Reply:
Nevermind, [2770beb](https://github.com/NotNite/Linkpearl/pull/2/commits/2770beb216cd3ecd28c1996924bf8087bd8e4947) resolves this.